app/tests/test_proxy.py

```diff
@@ -26,19 +26,31 @@
 import responses
 
 from .. import app
+from ..auth.oauth_redis import OAuthRedis
+from ..auth.oauth_client import RenkuWebApplicationClient
+from ..auth.oauth_provider_app import OAuthProviderApp
+from ..auth.utils import get_redis_key_from_token
+from ..auth.gitlab_auth import GL_SUFFIX
 from .test_data import (
     GITLAB_ISSUES,
     GITLAB_PROJECTS,
     PRIVATE_KEY,
     PUBLIC_KEY,
     TOKEN_PAYLOAD,
+    FERNET_KEY,
+    PROVIDER_APP_DICT,
 )
 
 
+# TODO: Completetly refactor all tests, masively improve test coverage.
```

Comment:
Love the comment, can we convert it into the ticket? Or if we have ticket for it already, just put the reference in the code.